### PR TITLE
XIVY-11980 implement VariableDetail component

### PR DESCRIPTION
**Pull request by ivy-lgi**
image

### CODE REVIEW

File: packages/new-editor/src/components/editor/variables/detail/Variable.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+export const VariableDetail = ({ variable }: VariableProps) => {
+  const metadataOptions = Object.keys(Metadata)
+    .filter(key => isNaN(Number(key)))
+    .map(key => {
+      const option = key.toString();
+      return { value: option, label: option };
+    });
+  metadataOptions.unshift({ value: 'None', label: 'None' });
+  const selectedMetadataOption = variable.metadata === undefined ? 'None' : Metadata[variable.metadata];
```
Comment (ivy-lgi) [inline packages/new-editor/src/components/editor/variables/detail/Variable.tsx:16]: I don't like this enum handling stuff and having to introduce an extra option to not select anything. Any ideas on improvements?

File: packages/new-editor/src/components/editor/variables/detail/Variable.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  return (
+    <>
+      <ToolbarTitle>Variable Configuration</ToolbarTitle>
```
Comment (ivy-lli) [inline packages/new-editor/src/components/editor/variables/detail/Variable.tsx:20]: why not sidebar header?

File: packages/new-editor/src/components/editor/variables/detail/Variable.tsx (hunk 1 above)
Comment (ivy-lli) [reply packages/new-editor/src/components/editor/variables/detail/Variable.tsx:16]: yes enums technially do not exist in ts. or better they really do not exist in js. ts does some "magic" here to fake them in js...

I would suppose something like in the inscription-view:
```ts
export const RESPONSIBLE_TYPE = {
  ROLE: 'Role',
  ROLE_FROM_ATTRIBUTE: 'Role from Attr.',
  USER_FROM_ATTRIBUTE: 'User from Attr.',
  DELETE_TASK: 'Nobody & delete'
} as const satisfies Record<WfActivatorType, string>;

Object.entries(RESPONSIBLE_TYPE)
        .filter(([value]) => !(props.optionFilter && props.optionFilter.includes(value as WfActivatorType)))
        .map(([value, label]) => ({ label, value }));
```

or in the form-editor:
```ts
const typeOptions: FieldOption<InputType>[] = [
  { label: 'Email', value: 'EMAIL' },
  { label: 'Password', value: 'PASSWORD' },
  { label: 'Text', value: 'TEXT' }
] as const;
```

at the moment you only "fake" the metas, so totally fine to hold this type as simple as possible.

**Review by ivy-lli — APPROVED** [review]

File: packages/new-editor/src/components/editor/variables/detail/Variable.tsx (hunk 2 above)
Comment (ivy-lgi) [reply packages/new-editor/src/components/editor/variables/detail/Variable.tsx:20]: This is the title underneath the header, not the header itself:
image

Although, maybe I should not use the `ToolbarTitle` component for this?

**Review by ivy-lli — APPROVED** [review]

**Pull request merged**